sru/release-29/test-key-rename-failure.sh

```diff
@@ -0,0 +1,98 @@
+sleep 5
+
+# Install latest ubuntu-advantage-tools
+lxc exec $name -- apt-get update > /dev/null
+lxc exec $name -- apt-get install  -y ubuntu-advantage-tools > /dev/null
+echo -e "\n* Latest u-a-t is installed"
+echo "###########################################"
+lxc exec $name -- apt-cache policy ubuntu-advantage-tools
+echo -e "###########################################\n"
+
+# Attach so we can have some gpg keys in place
+lxc exec $name -- pro attach $token
+echo -e "\n* Pro is attached"
+echo "###########################################"
+lxc exec $name -- pro status --wait
+echo -e "###########################################\n"
+
+# Upgrade u-a-t to new version
+# ----------------------------------------------------------------
+if [ $install_from == 'staging' ]; then
+  lxc exec $name -- sudo add-apt-repository ppa:ua-client/staging -y > /dev/null
+  lxc exec $name -- apt-get update > /dev/null
+  lxc exec $name -- apt-get install ubuntu-advantage-tools -y > /dev/null
+elif [ $install_from == 'proposed' ]; then
+  lxc exec $name -- sh -c "echo \"deb http://archive.ubuntu.com/ubuntu $series-proposed main\" | tee /etc/apt/sources.list.d/proposed.list"
+  lxc exec $name -- apt-get update > /dev/null
+  lxc exec $name -- apt-get install ubuntu-advantage-tools -y > /dev/null
+else
+  lxc file push $install_from $name/new-ua.deb
+  lxc exec $name -- dpkg -i /new-ua.deb > /dev/null
+fi
+# ----------------------------------------------------------------
+
+echo -e "\n* Renaiming keys back to their original name"
+echo "###########################################"
+lxc exec $name -- mv /etc/apt/trusted.gpg.d/ubuntu-pro-esm-infra.gpg /etc/apt/trusted.gpg.d/ubuntu-advantage-esm-infra-trusty.gpg
+lxc exec $name -- mv /etc/apt/trusted.gpg.d/ubuntu-pro-esm-apps.gpg /etc/apt/trusted.gpg.d/ubuntu-advantage-esm-apps.gpg
+echo -e "###########################################\n"
+
+echo -e "\n* List gpg keys"
+echo "###########################################"
+lxc exec $name -- ls /etc/apt/trusted.gpg.d/
+echo -e "###########################################\n"
+
+# Modify the postinst script to fail after renaming just one key
+lxc exec $name -- sed -i "s/^\s\+SERVICES=.*/error/g" /var/lib/dpkg/info/ubuntu-advantage-tools.postinst
+echo -e "\n* Running postinst script again"
+echo "###########################################"
+lxc exec $name -- /var/lib/dpkg/info/ubuntu-advantage-tools.postinst configure || true
+echo -e "###########################################\n"
+
+# Check that only one key got renamed
+echo -e "\n* List gpg keys"
+echo "###########################################"
+lxc exec $name -- ls /etc/apt/trusted.gpg.d/
+echo -e "###########################################\n"
+
+# Make sure the services are still enabled
+echo -e "\n* Check that services are still enabled"
+echo "###########################################"
+lxc exec $name -- pro status --wait
+echo -e "###########################################\n"
+
```

Comment:
At this point, I think it is useful to run `apt update` and show that no error regarding the signing keys is presented